### PR TITLE
Box: adjust default padding if in primary slot of Split

### PR DESCRIPTION
**Pull request by AquiGorka**

### CODE REVIEW

**Comment by bpierre** [comment]
This is quite the change! Is it because of the padding of the large box in a split?

**Comment by AquiGorka** [comment]
I recalled we agreed this would be the default padding no? It is the padding for App Center and Voting's Boxes

**Comment by bpierre** [comment]
But only for the big ones right? In the `primary` slot of a `Split`. I think we can change this default only in that case.

File: src/components/Split/Split.js [hunk 1]
```diff
@@ -1,31 +1,38 @@
...
   const oneColumn = layout === 'small' || layout === 'medium'
   return (
-    <div
-      css={`
-        display: ${oneColumn ? 'block' : 'flex'};
-        padding-bottom: ${3 * GU}px;
-      `}
-    >
-      <div css="flex-grow: 1">{primary}</div>
+    <Inside name="Split">
```
Comment (sohkai) [inline src/components/Split/Split.js]: Wasn't really sure if we needed all these levels, but I thought we might as well for consistency's sake?

Comment (bpierre) [reply src/components/Split/Split.js]: Yes it can be useful!

Comment (bpierre) [reply src/components/Split/Split.js]: Yes it can be useful!

**Pull request merged**